### PR TITLE
Take calls from plain php files into account

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: tests/Rules/UnusedPublicClassMethodRule/UnusedPublicClassMethodRuleTest.php [hunk 1]
```diff
@@ -47,16 +48,23 @@ public static function provideDataTests(): Iterator
...
         );
+        $errorMessage2 = sprintf(
+            UnusedPublicClassMethodRule::ERROR_MESSAGE,
+            MethodForTests::class,
+            'calledStaticCall'
+        );
         yield [[
             __DIR__ . '/Fixture/Tests/MethodForTests.php',
             __DIR__ . '/Fixture/Tests/SkipCalledInTests.php',
-        ], [[$errorMessage, 13, RuleTips::SOLUTION_MESSAGE]]];
```
Comment (staabm) [inline tests/Rules/UnusedPublicClassMethodRule/UnusedPublicClassMethodRuleTest.php:59]: I think this test was wrong before and reporting 2 errors is correct, as the only calls are in Test-classes

**Review by samsonasik — APPROVED** [review]

**Comment by staabm** [comment]
ping @TomasVotruba would be great to get this false positives out of the way.

please also see the 2 other PRs
- https://github.com/TomasVotruba/unused-public/pull/80
- https://github.com/TomasVotruba/unused-public/pull/81

File: src/Collectors/MethodCall/MethodCallCollector.php [hunk 2]
```diff
@@ -50,13 +50,10 @@ public function processNode(Node $node, Scope $scope): ?array
             return null;
         }
 
-        $classReflection = $scope->getClassReflection();
-        if (! $classReflection instanceof ClassReflection) {
```
Comment (staabm) [inline src/Collectors/MethodCall/MethodCallCollector.php:54]: before this PR, this check made collection on code which is not within class-scope early exiting.

**Comment by staabm** [comment]
@TomasVotruba just one more ;)

**Comment by staabm** [comment]
@TomasVotruba if you could merge this one and tag a new release, we would have fixed a major source of false postives with this lib 🤘

**Comment by TomasVotruba** [comment]
Let's ship this :)  thank you :clap:

**Comment by staabm** [comment]
thank you

**Pull request merged**